apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/TableStatsCollectionSparkApp.java

```diff
@@ -43,10 +47,16 @@ protected void publishStats(IcebergTableStats icebergTableStats) {
   public static void main(String[] args) {
     List<Option> extraOptions = new ArrayList<>();
     extraOptions.add(new Option("t", "tableName", true, "Fully-qualified table name"));
+    extraOptions.add(
+        new Option("s", "skipStorageStats", false, "Whether to skip storage stats collection"));
+
     CommandLine cmdLine = createCommandLine(args, extraOptions);
     TableStatsCollectionSparkApp app =
         new TableStatsCollectionSparkApp(
-            getJobId(cmdLine), createStateManager(cmdLine), cmdLine.getOptionValue("tableName"));
+            getJobId(cmdLine),
+            createStateManager(cmdLine),
+            cmdLine.getOptionValue("tableName"),
+            cmdLine.hasOption("skipStorageStats"));
```

Comment:
How does the caller (scheduler?) determine when to skip? If it's ad-hoc, then caller knows and might as well not run at all, if the caller is scheduler, then we have filter assigned to the task where the table can be skipped, is this for another use case?

Reply:
I want to keep it disabled for all tables now and re-enable it when we have a better way to handle the storage part. Also apart from scheduler, when we run adhoc jobs, it might be better to have this enabled to collect storage stats as required.

Reply:
would this work
`kubectl patch cronjobs <job-name> -p '{"spec" : {"suspend" : true }}'`

Reply:
IIUC, This implies suspending the whole job. But this PR targets to disable only storage stats computation. Other table stats , like metadata and iceberg stats and current snapshot metrics will still be computed.